packages/release-please/src/release-please.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Application } from 'probot';
+import { GitHubAPI } from 'probot/lib/github';
+import { ReleaseType } from 'release-please/build/src/release-pr';
```

Comment:
Reaching down into non-index exported classes like this is generally something we want to avoid.  We should make sure any methods or classes we want to use are exported from the root @bcoe

Reply:
Filed https://github.com/googleapis/release-please/issues/249 and added a TODO once the types are exposed.